src/Components/Blazor/testassets/MonoSanityClient/MonoSanityClient.csproj

```diff
@@ -11,7 +11,4 @@
     <!-- loader.js is hard-coded to assume it can load .pdbs regardless of Debug/Release configuration -->
     <BlazorEnableDebugging>true</BlazorEnableDebugging>
   </PropertyGroup>
-
-  <ItemGroup>
-  </ItemGroup>
```

Comment:
this was my favorite item group. F to pay respects

Reply:
Pity it wasn't conditional. Then again I've always been partial to `<ItemGroup />`